### ISSUE TITLE
Roadmap planning

### ISSUE THREAD

**Issue by Sytten** [plangrid/flask-rebar#151]
Calling the major devs @RookieRick @airstandley @barakalon.
Could we have a roadmap call to establish a roadmap for the project? I would like to contribute, but I am not sure where/what is a priority going forward.
I have a seen a lot of people moving to fastapi lately and I want to imitate the way the dev engages with the community (aka I want to redo the doc, add typing and make more examples to get started).

**Comment by RookieRick** [comment]
Sounds like a great idea and we've got a couple other folks on the team here who would be interested;  I'll quiz people here on preferred timing and set something up 👍

**Comment by Sytten** [comment]
Can we have access to the notes of the meeting?

**Comment by RookieRick** [comment]
Absolutely! I just gotta transcribe them (which is on my imminent to-do list I promise :joy: ) and I'll add them to our docs folder
